crt_portal/cts_forms/templates/forms/pro_template.html

```diff
@@ -23,16 +23,17 @@ <h2 id="intake">Intake</h2>
   </div>
 </div>
 
-<div class="crt-portal-card">
+<div class="crt-portal-card crt-portal-card--proform">
   <div class="crt-portal-card__content crt-portal-card__content--lg">
-    <a name="contact"></a>
```

Comment:
`name` is deprecated, and all of the others were getting targeted via ID already, so just changed this to match!